src/dosext/mfs/mfs.c

```diff
@@ -3473,12 +3473,12 @@ static int dos_fs_redirect(struct vm86_regs *state, char *stk)
       set_32bit_size_or_position(&_sft_position(sft), f->seek);
       if (ret + s_pos > sft_size(sft)) {
         /* someone else enlarged the file! refresh. */
-        fstat(f->fd, &f->st);
```

Comment:
I'd rather add assert.
fstat() is not supposed to fail, but
it doesn't mean we need to quietly
ignore if it ever does.

Reply:
okay, done